static/src/javascripts/projects/commercial/modules/header-bidding/types.js

```diff
@@ -97,7 +97,8 @@ export type PrebidBidder = {
         | PrebidOpenXParams
         | PrebidOzoneParams
         | PrebidAdYouLikeParams
-        | PrebidPubmaticParams,
+        | PrebidPubmaticParams
+    >,
```

Comment:
Enforcing `bidParams` to always return a Promise seemed the cleanest approach, but there are still many instances where it could remain synchronous.

Having a mix of types, as below, seemed worse and harder to reason about:

```
        | Promise<PrebidSonobiParams>
        | PrebidIndexExchangeParams
        | PrebidTrustXParams
        | Promise<PrebidOpenXParams>
        | PrebidOzoneParams
        | Promise<PrebidAdYouLikeParams>
        | PrebidPubmaticParams
```

Reply:
yeah def - if it needs to be async in one instance, all should be imo

Reply:
Agree. The only other option I can think of is bidParams and bidParamsAsync, and the consumer checks both (or only one if it knows which it needs)